src/Covenant/Index.hs

```diff
@@ -1,8 +1,14 @@
 {-# LANGUAGE RoleAnnotations #-}
 
--- | Module: Covenant.Index
+-- |
+-- Module: Covenant.Index
+-- Copyright: (C) MLabs 2025
+-- License: Apache 2.0
+-- Maintainer: koz@mlabs.city, sean@mlabs.city
 --
--- Positional indexes, starting from 0.
+-- Positional indexes, starting from 0, and cardinality indicators.
```

Comment:
I thought it was supposed to be 'indices' but I guess grammarians are conflicted about whether that's indisputably correct

Reply:
I'm not too sure about that - I've seen both. We can always change it if someone isn't happy.